newswires/client/src/sharedTypes.ts

```diff
@@ -44,8 +44,12 @@ export type WiresQueryResponse = z.infer<typeof WiresQueryResponseSchema>;
 
 export const QuerySchema = z.object({
 	q: z.string(),
-	supplier: z.array(z.string()),
-	subject: z.array(z.string()),
+	supplier: z.array(z.string()).optional(),
+	supplierExcl: z.array(z.string()).optional(),
+	keywords: z.ostring(),
```

Comment:
I wonder whether we'd want to have these as arrays of strings too, further down the line, in order to make it easier to handle them in the UI? But not necessary at this stage afaict

Reply:
yep - broadly I think depends whether we want to standardise on `&keyword=a&keyword=b&keyword=c` or `&keywords=a,b,c` - both fine but we should pick one and not do both... later